apps/pie-docs/package.json

```diff
@@ -11,9 +11,8 @@
   "scripts": {
     "build": "DEPLOYMENT_ENVIRONMENT='production' eleventy",
     "build:dev": "DEPLOYMENT_ENVIRONMENT='development' eleventy",
-    "clean:dist": "run -T rimraf dist",
-    "cz": "run -T cz",
-    "dev": "yarn clean:dist && npx @11ty/eleventy --serve",
+    "clean": "run -T rimraf dist",
```

Comment:
Aligned with naming convention from other packages, which also allows this to be run from the root